sdks/python/apache_beam/io/avroio.py

```diff
@@ -146,6 +153,10 @@ def __init__(self, file_pattern=None, min_bundle_size=0, validate=True,
       use_fastavro (bool); when set, use the `fastavro` library for IO, which
         is significantly faster, and will likely become the default
     """
+    if sys.version_info[0] >= 3 and not use_fastavro:
+      warnings.warn("Due to a known issue in avro-python3 package, it is "
+                    "recommended to use fastavro with Beam Avro IO on "
```

Comment:
Please prioritize fixing this so that this kinds of warnings can be removed (can be after submitting this PR).

Reply:
AFAIK, the failure is fixed in Avro HEAD, and we are awaiting for Avro release.

Reply:
Looks like Avro release might happen soon [1]: 
@fredo838, could you please check whether currenlty-disabled-on-python3 test passes if we install: https://dist.apache.org/repos/dist/dev/avro/avro-1.9.0-rc2/py3/avro-python3-1.9.0.tar.gz

You can pass that file as --extra_package to a Dataflow job.

Thank you.
[1] https://lists.apache.org/thread.html/ff724cff8fed12ebc8316e0dc1e876f0ca8080ab68b6165b7f109665@%3Cdev.avro.apache.org%3E

Reply:
I will take a look.